UpdatedImageProcessing/Classifiers/__init__.py

```diff
@@ -0,0 +1,5 @@
+#from classify_color import ColorClassifier
+#from classify_shape import ShapeClassifier
+#from classify_tester import ClassifierTester
```

Comment:
Why are these imports commented out?

Reply:
For testing purposes. I committed this to save in progress work. I uncommented these lines in https://github.com/FlintHill/SUAS-Competition/pull/97/commits/bfde9a153d5b899db8812b478525ed8542de8f28 & https://github.com/FlintHill/SUAS-Competition/pull/97/commits/a81de5d170d31a737251567f72ec9961d3b3bd9e